rsmtool/configuration_parser.py

```diff
@@ -1235,7 +1233,18 @@ def validate_config(cls, config, context='rsmtool'):
                     raise ValueError("Invalid SKLL objective. Please refer to the SKLL "
                                      "documentation and choose a valid tuning objective.")
 
-        # 9. Check that if we are running rsmtool to ask for
+        # 9. Check if that if "skll_fixed_parameters" is specified,
+        # it's specified for SKLL model and _not_ a built-in linear
+        # regression model; we cannot check whether the parameters
+        # are valid at parse time but SKLL will raise an error#
+        # at run time for any invalid parameters
+        if new_config['skll_fixed_parameters']:
```

Comment:
Should this be an error to make sure the user doesn't ignore the warning?

Reply:
We raise a warning for `skll_objective` too so I just followed the same logic here.

Reply:
Makes sense!